connect/src/main/java/com/stripe/android/connect/webview/StripeConnectWebViewContainer.kt

```diff
@@ -208,8 +214,22 @@ internal class StripeConnectWebViewContainerImpl(
                 null
             )
         }
+
+        override fun shouldOverrideUrlLoading(view: WebView?, request: WebResourceRequest?): Boolean {
+            view ?: return false // the view shouldn't be null, but if it is then don't handle the request
```

Comment:
You should be able to make the params non-null like so:
```suggestion
        override fun shouldOverrideUrlLoading(view: WebView, request: WebResourceRequest): Boolean {
```
It's just when you have Android Studio generate the Kotlin override function signature for a Java class and `@NonNull` isn't specified, then it assumes nullable by default.

Reply:
This is me being paranoid because I'm very skeptical anytime `@NonNull` isn't specified lol. I'm fine with this though